src/renderer/api-decorator.js

```diff
@@ -415,6 +445,61 @@ limitations under the License.
         });
     }
 
+    //https://developer.mozilla.org/en-US/docs/Web/API/Window/open
+    //All features can be set to yes or 1, or just be present to be "on". Set them to 'no' or 0, or in most cases just omit them, to be "off".
+    function featureToBool(value = '') {
+        switch (value.toLowerCase()) {
+            case '0':
+                return false;
+            case 'no':
+                return false;
+            default:
+                return true;
+        }
+    }
+
+    //We need to do this as all values are text and convertToElectron does not handle type changes only name translation.
+    function featuresToOptionsObj(features) {
+        features = features.split(' ').join('');
+        let featuresObj = {};
+        features.split(',').forEach(item => {
```

Comment:
:neckbeard: function would be much shorter if you reduced the array into featuresObj with returns in the switch statement.